content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -202,12 +202,13 @@ In releases older than Debian 12 and Ubuntu 22.04, folder `/etc/apt/keyrings` do
    echo 'deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/ /' | sudo tee /etc/apt/sources.list.d/kubernetes.list
    ```
 
-4. Update the `apt` package index, install kubelet, kubeadm and kubectl, and pin their version:
+4. Update the `apt` package index, install kubelet, kubeadm and kubectl, and pin their version and enable kubelet to ensure it's automatically started on startup::
 
    ```shell
    sudo apt-get update
    sudo apt-get install -y kubelet kubeadm kubectl
    sudo apt-mark hold kubelet kubeadm kubectl
+   sudo systemctl enable --now kubelet
```

Comment:
isn't it automatically enabled if using a package?

Reply:
i see kubeadm throws an error if the kubelet service is not enabled
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/preflight/checks.go#L140

should the package enable it?
if so, that should be PRed in the kubernetes/release repository.

Reply:
xref
- https://github.com/kubernetes/kubeadm/issues/2178

Reply:
@tuladhar @stmcginnis
so reading the above discussion i remembered some details. we should not enable the service as part of the docs.
it should be enabled by kubeadm or the user.

Reply:
In installation section of Red Hat-based distro ([here](https://github.com/kubernetes/website/blob/9c55a6e15f8aada2b1db46b5dd1cf40024c2d655/content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md?plain=1#L262)) and Without a package manager ([here](https://github.com/kubernetes/website/blob/9c55a6e15f8aada2b1db46b5dd1cf40024c2d655/content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md?plain=1#L322)), I see `kubelet` service enable is mentioned in both places.

Should we avoid mentioning here as well?

Reply:
i think there should be a comment about an optional 'enable' step in all places, not a required step

Reply:
Yeah, I have made the changes in this commit 25f1e8e6815efe030aaaa73f16d0dfd256e1ee33. Please check.